lib/generators/sufia/templates/workflow.json.erb

```diff
@@ -0,0 +1,60 @@
+{
+  "workflows": [
+    {
+      "name": "default",
+      "label": "Digital collections workflow",
+      "description": "A multi-step workflow for digital collections",
+      "actions": [
+         {
+           "name": "finalize_digitization",
```

Comment:
Perhaps the thing to do here is to make a change in CC to have this in the default workflow:
           "notifications": [
             {
               "notification_type": "email",
               "name": "Sufia::Workflow::ConfirmSubmission",
               "to": ["creating_user"],
               "cc": ["advising"]
             }

Then to test this issue, I could load the CC default work. If this makes sense, I'll write an issue and put a pull request for it. Let me know.